web/src/LogStore.ts

```diff
@@ -736,5 +760,8 @@ const logStoreContext = React.createContext<LogStore>(new LogStore())
 export function useLogStore(): LogStore {
   return useContext(logStoreContext)
 }
+export function useLogAlertIndex(): LogAlertIndex {
```

Comment:
Good question! Mostly to help with testing. FakeAlertIndex is way simpler than trying to support the full LogStore API.

I played around with having a separate AlertIndex (that LogStore owns and keeps up-to-date). Making AlertIndex an interface made the overall PR smaller. But it might still make sense to separate them in the future.

Reply:
ah, ok! i think that makes sense. it might be worth adding a comment above `useLogAlertIndex` to explain why/where you'd use it instead of `useLogStore`.

Reply:
comments added!